MicroProfile-Fault-Tolerance/tck-runner/.m2/extensions.xml

```diff
@@ -0,0 +1,11 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+
+    <extension>
+        <groupId>fish.payara.maven.extensions</groupId>
+        <artifactId>regex-profile-activator</artifactId>
+        <version>0.2</version>
```

Comment:
As per our call I think this needs to be removed and the version in `.mvn/extensions.xml` needs replacing with 0.2. Let me know if this causes the same issues.

Reply:
Did move the extension to `.mvn` where it is picked up but this still has the behaviour of ending up with MP 1.1 in case I do not specify a version using `-Dpayara.version`. Need to double check which version is default.

Reply:
Ok, default version was `5.184` (somewhat surprisingly for me). That explains that.